### DISCUSSION TITLE
Use Tab to indent bullets on Markdown Alternative

### DISCUSSION THREAD

**Discussion by maximiliaan33** [standardnotes/forum#2676] [Suggestions]
Hi there,

I'm a big fan of Standard Note. I'm mostly using Markdown Alternative editor as it looks clean.

The only problem I have is that when I use a bullet list, I can't make indents with the keyboard (using tab normally).

I have to use the button in the app to make an indent, which feels very cumbersone.

Can tab be used for making indents in bulleted lists?

It's both on the application and in the browser.

Best,

Max

**Comment by effieeee** [comment]
I'm afraid that tab really can't be used to indent and make sub bullets at the moment :( But I'll take note of this as a needed enhancement on Markdown Alternative
